hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/common/CalcIntoScanPushdownRule.java

```diff
@@ -33,61 +36,66 @@
 import java.util.List;
 
 import static com.hazelcast.jet.sql.impl.opt.Conventions.LOGICAL;
+import static java.util.Arrays.asList;
+import static org.apache.calcite.rex.RexUtil.composeConjunction;
 
 /**
  * Logical rule that pushes a {@link Calc} down into a {@link TableScan} to allow for constrained scans.
  * See {@link HazelcastTable} for more information about constrained scans.
  * <p>
  * Before:
  * <pre>
- * LogicalCalc[filter=exp1]
- *     LogicalScan[table[filter=exp2]]
+ * Calc[filter=exp1]
+ *     TableScan[table[filter=exp2]]
  * </pre>
  * After:
  * <pre>
- * LogicalScan[table[filter=exp1 AND exp2]]
+ * TableScan[table[filter=exp1 AND exp2]]
  * </pre>
  */
 @Value.Enclosing
-public final class CalcIntoScanLogicalRule extends RelRule<Config> implements TransformationRule {
+public final class CalcIntoScanPushdownRule extends RelRule<Config> implements TransformationRule {
```

Comment:
`CalcIntoScanPushdownRule` -> `CalcIntoScanRule`
"calc into scan" is clear enough. If we stick with "push down", it should be `PushDown`, not `Pushdown`.